publication-commons/src/main/java/no/unit/nva/publication/model/business/Resource.java

```diff
@@ -166,6 +167,13 @@ public List<File> getFiles() {
                    .toList();
     }
 
+    @JsonIgnore
+    public Optional<File> getFileByIdentifier(UUID identifier) {
```

Comment:
compared to this other methis in this same file, probably more or less the same thing?
public Optional<FileEntry> getFileEntry(SortableIdentifier identifier)

Reply:
Yes, but when we are in context of file, I do like this:

`resource.getFileByIdentifier(file.getIdentifier());`

instead of doing:

`resource.getFileEntry(new SortableIdentifier(file.getIdentifier().toString())).map(FileEntry::getFile)`

I think it is ugly to convert UUID to string and then to SortableIdentifier :)